internal/budgetadjustments/flags/flags.go

```diff
@@ -0,0 +1,95 @@
+package flags
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/nobl9/sloctl/internal/csv"
+)
+
+const (
+	FlagFile       = "file"
+	FlagDryRun     = "dry-run"
+	FlagAdjustment = "adjustment-name"
+	FlagFrom       = "from"
+	FlagTo         = "to"
+	FlagSloProject = "slo-project"
+	FlagSloName    = "slo-name"
+)
+
+func MustNotifyDryRunFlag() {
+	if _, err := fmt.Fprintln(os.Stderr, "Running in dry run mode, changes will not be applied."); err != nil {
+		panic(err)
+	}
+}
+
+func MustRegisterFileFlag(cmd *cobra.Command, storeIn *string) {
+	cmd.Flags().StringVarP(storeIn, FlagFile, "f", "",
+		"File path, glob pattern or a URL to the configuration in YAML or JSON format.")
+	if err := cmd.MarkFlagRequired(FlagFile); err != nil {
+		panic(err)
+	}
+}
+
+func RegisterDryRunFlag(cmd *cobra.Command, storeIn *bool) {
+	cmd.Flags().BoolVarP(storeIn, FlagDryRun, "", false,
+		"Submit server-side request without persisting the configured resources.")
+}
+
+func MustRegisterOutputFormatFlags(
+	cmd *cobra.Command,
+	outputFormat, fieldSeparator, recordSeparator *string,
+) {
+	cmd.PersistentFlags().StringVarP(outputFormat, "output", "o", "yaml",
+		`Output format: one of yaml|json|csv.`)
+
+	cmd.PersistentFlags().StringVarP(fieldSeparator, csv.FieldSeparatorFlag, "",
+		csv.DefaultFieldSeparator, "Field Separator for CSV.")
+
+	cmd.PersistentFlags().StringVarP(recordSeparator, csv.RecordSeparatorFlag, "",
+		csv.DefaultRecordSeparator, "Record Separator for CSV.")
```

Comment:
we shouldn't be redeclaring flags which are already used in sloctl, see https://github.com/nobl9/sloctl/blob/main/internal/flags.go. Not all of these flags that are already defined there should be shared, but some do, maybe we can extract these to a separate package?